GoogleForCreators/web-stories-wp
# Editor: Fix unexpected shift of items in floating menu

## Context

UI elements in the contextual floating edit menu would unexpectedly horizontally resize based on width of selected font.

## Summary

These elements no longer resize.

## Relevant Technical Choices

Some elements in this menu had a style rule of `width: 100%`. The affected elements were in a flexbox layout, so setting `width: 100%` forced them to fill all available horizontal space, whether or not this was needed for proper styling. This style was changed to `width: auto` to prevent this behavior.

## To-do

- Not directly related to these changes, the color picker needs to be refactored to better handle RTL layout and simplify the code.

## User-facing changes

Before - items change widths (notably the 'More' button):
https://user-images.githubusercontent.com/4173034/176249662-848bfc9e-23ca-4ee7-9d1a-c36c191d16f8.mov

After - items retain widths (note: the menu itself shows correct behavior of resizing when the color picker changes)
https://user-images.githubusercontent.com/4173034/176249969-c1fbf85a-fefe-41d3-bb10-d2575ae2d1be.mov


## Testing Instructions

- Select a text element on the editor canvas
- Change the selected font
- Items in the menu should retain width




## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #11779
